tests/deploy/values/ci-eks-config-1.yml

```diff
@@ -8,6 +8,7 @@ image:
 
 cloudbeat:
   deploy: false
+  repository: 704479110758.dkr.ecr.eu-west-2.amazonaws.com/cloudbeat
```

Comment:
is this the right account id?

Reply:
Yes, this is currently needed for backward compatibility